src/testUtils.ts

```diff
@@ -0,0 +1,50 @@
+export default {
+  runAsync: (fn) => {
+    return new Promise((resolve, reject) => {
+      setTimeout(() => {
+        try {
+          resolve(fn())
+        }
+        catch (err) {
+          reject(err)
+        }
+      }, 1)
+    })
+  },
+  runSequentialAsync: (...fns) => {
+    return new Promise((resolve, reject) => {
+      setTimeout(() => {
+        try {
+          let i = 0
+          for (; i < fns.length - 1; i++) {
+            fns[i]()
+          }
+          resolve(fns[i]())
+        }
+        catch (err) {
+          reject(err)
+        }
+      }, 1)
+    })
+  },
+  runParallelAsync: (...fns) => {
+    const promises: Promise<any>[] = []
```

Comment:
Can simplify this by
```ts
fns.map(fn => runSync(fn))
```

You can also export each util function individually using named export. That's typically a better way to go than default export:

```ts
export function runAsync() ...
export function runParallelAsync() ...

// AssertOrder.spec.ts
import { runAsync, runParallelAsync } from './testUtil'
// or
import * as testUtil from './testUtil'
```